authorship/vocalize_data.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import pandas as pd
+
+
+class VocalizeDataSource:
```

Comment:
Maybe a docstr here to describe the expected file format?

Reply:
done